DEVFILE_REGISTRY.md

```diff
@@ -61,3 +61,171 @@ If you want to send telemetry information to your own Segment instance, specify
     key: <your-segment-write-key>
     registryViewerWriteKey: <your-segment-write-key>
 ```
+
+## Using Specific Container Images
+
+By default, the operator deploys a Devfile Registry using a Pod containing three containers:
+
+- `devfile-registry` container runs the API serving the Devfile stacks. The container image used by default
+is `quay.io/devfile/devfile-index:next` and you can change it with the field `spec.devfileIndex.image`.
+- `oci-registry` container runs a standard OCI registry, serving the stacks in the OCI format.
+The container image used by default is `quay.io/devfile/oci-registry:next` and you can change it with the field `spec.ociRegistry.image`.
+- `registry-viewer` container runs a web frontend, to help the user brwose the Devfile stacks.
+The container image used by default is `quay.io/devfile/registry-viewer:next` and you can change it with the field `spec.registryViewer.image`.
+
+```bash
+$ cat <<EOF | oc apply -f -
+apiVersion: registry.devfile.io/v1alpha1
+kind: DevfileRegistry
+metadata:
+  name: my-devfile-registry
+spec:
+  devfileIndex:
+    image: quay.io/my-devfile/devfile-index:next
+  ociRegistry:
+    image: quay.io/my-devfile/oci-registry:next
+  registryViewer:
+    image: quay.io/my-devfile/registry-viewer:next
+EOF
+```
+
+### Defining the ImagePullPolicy for pulling containers
+
+By default, the containers will be pulled depending on the policy set on the Kubernetes or OpenShift cluster the registry is deployed on.
+
+You can set a specific pull policy for each container, by setting the fields `spec.devfileIndex.imagePullPolicy`, `spec.ociRegistry.imagePullPolicy` and `spec.registryViewer.imagePullPolicy` to a value `IfNotPresent`, `Always` or `Never`. See the [Kubernetes documentation](https://kubernetes.io/docs/concepts/containers/images/#image-pull-policy) for more information.
+
+
+```bash
+$ cat <<EOF | oc apply -f -
+apiVersion: registry.devfile.io/v1alpha1
+kind: DevfileRegistry
+metadata:
+  name: my-devfile-registry
+spec:
+  devfileIndex:
+    image: quay.io/my-devfile/devfile-index:next
+    imagePullPolicy: Always
+  ociRegistry:
+    image: quay.io/my-devfile/oci-registry:next
+    imagePullPolicy: Always
+  registryViewer:
+    image: quay.io/my-devfile/registry-viewer:next
+    imagePullPolicy: Always
+EOF
+```
+
+## Disabling the web frontend 
+
+You can ask the operator to deploy the Devfile Registry without the `registry-viewer` container, by setting the field `spec.headless` to `true`.
+
+```bash
+$ cat <<EOF | oc apply -f -
+apiVersion: registry.devfile.io/v1alpha1
+kind: DevfileRegistry
+metadata:
+  name: headless-devfile-registry
+spec:
+  devfileIndex:
+    image: quay.io/devfile/devfile-index:next
+  telemetry:
+    registryName: test
+  headless: true
+EOF
+```
+
+## Using a Persistent Volume to store OCI images
```

Comment:
As part of https://github.com/devfile/api/issues/1093, we will be retiring the storage field for the devfile registry, as it's no longer used. So we should leave it out of the documentation